clients/bigquery/bigquery.go

```diff
@@ -49,7 +49,7 @@ func (s *Store) PrepareTemporaryTable(tableData *optimization.TableData, tableCo
 			Mode:              tableData.Mode(),
 		}
 
-		if err := ddl.AlterTable(tempAlterTableArgs, tableData.ReadOnlyInMemoryCols().GetColumns()...); err != nil {
+		if err := tempAlterTableArgs.Alter(tableData.ReadOnlyInMemoryCols().GetColumns()...); err != nil {
```

Comment:
I feel like dropping `Table` calling the function just `Alter` makes it hard to tell what is being altered, and now that it's a receiver function on an argument struct if I were seeing it for the first time I would assume that this code was just altering the `tempAlterTableArgs` struct, not creating a temporary table.

Reply:
Oh interesting. For me, I look at `ALTER` and automatically assume it's the SQL keyword for DDL. I can add `AlterTable` back to be more clear though